spec/Body/Body.vspec

```diff
@@ -161,11 +161,53 @@ Windshield.WasherFluid.Level:
 ##
 Lights:
   type: branch
-  description: All lights.
+  instances: ["Beam", "Running", "Backup", "Parking", "Brake", "Fog", "Hazard", "Indicator"]
+  description: Exterior lights.
 
-# Include lights specification and attach it to the Lights branch
-#include ExteriorLights.vspec Lights
+Lights.Beam:
+  type: branch
+  instances: ["Low","High"]
+  description: Beam lights
+# Include lights specification and attach it to the Lights.Beam branch
```

Comment:
Comment seems redundant a it describes the obvious from the code on the next line?